packages/v/v8/xmake.lua

```diff
@@ -49,36 +38,68 @@ package("v8")
     "managed": False,
     "custom_deps": {},
   }]]=])
+
         if package:is_plat("windows") then
-            envs.DEPOT_TOOLS_WIN_TOOLCAHIN = "0"
+            envs.DEPOT_TOOLS_WIN_TOOLCHAIN = "0"
+            envs.GYP_MSVS_VERSION = "2022"
         end
         local gclient = is_host("windows") and "gclient.bat" or "gclient"
-        os.vrunv(gclient, {"sync", "-v"}, {envs = envs})
+
+        -- Prevent long path issue on Windows
+        os.vrun("git config --global core.longpaths true")
+
+        -- Update repository and dependencies
+        -- Clean any local changes to apply patches
+        os.vrunv(gclient, {"sync", "--reset", "--delete_unversioned_trees", "-v"}, {envs = envs})
+
+        -- Fix GN issue
+        local patchPath = path.join(package:scriptdir(), "patches", "fix_chromium.patch")
```

Comment:
where is this patch file?

Reply:
And different versions of patches should be different, we need to handle them by version